summingbird-storm/src/main/scala/com/twitter/summingbird/storm/StormTopologyBuilder.scala

```diff
@@ -12,6 +13,25 @@ import org.apache.storm.generated.StormTopology
 import org.slf4j.LoggerFactory
 import scala.reflect.ClassTag
 
+import scala.collection.{ Map => CMap }
+
+object StormTopologyBuilder {
+  type Item[T] = (Timestamp, T)
+
+  type AggregatedKey[K] = (K, BatchID)
```

Comment:
AggregatedKey -> AggregateKey
iiuc we want to indicate key of an aggregate, rather than key that has been aggregated.